lib/experimental/Information/Tags/BalanceTag/index.tsx

```diff
@@ -26,7 +26,7 @@ export const BalanceTag = forwardRef<HTMLDivElement, Props>(
       <BaseTag
         ref={ref}
         className={cn(
-          "pl-1",
+          "line-clamp-1 flex flex-row items-center justify-between gap-0.5 rounded-md py-0.5 pl-1 pr-2 text-base font-medium",
```

Comment:
Why are these changes needed?

Reply:
I think is a bad conflict resolution, let me put it as before

Reply:
Perfect, thanks!